src/coffee/sync/category-sync.coffee

```diff
@@ -0,0 +1,21 @@
+_ = require 'underscore'
+BaseSync = require './base-sync'
+CategoryUtil = require './utils/category-utils'
+
+###*
+ * CategorySync Sync class
+###
+class CategorySync extends BaseSync
+
+  constructor: ->
+    # Override base utils
+    @_utils = new CategoryUtil()
+
+  _doMapActions: (diff, new_obj, old_obj) ->
```

Comment:
Is not required, but this is the method definition. It's up to the actual utility functions to use it or not